nbs/tutorials/methods.ipynb

```diff
@@ -11,10 +11,43 @@
    "cell_type": "markdown",
```

Comment:
<p>I think this table is wrong. For example, DiverseCF is a non-parametric methods. Check out the readme page for the correct classification.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Corrected the table